packages/rx_bloc_cli/mason_templates/bricks/feature_otp/__brick__/lib/feature_otp/fake_classes/fake_text_field_validator.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:widget_toolkit/text_field_dialog.dart';
+
+class FakeTextFieldValidator extends TextFieldValidator<String> {
```

Comment:
Also here you can user `OtpTextFieldValidator` as a name, instead of `FakeTextFieldValidator`